contracts/stellar-interchain-token-service/src/interface.rs

```diff
@@ -365,4 +365,17 @@ pub trait InterchainTokenServiceInterface:
         metadata: Option<Bytes>,
         gas_token: Option<Token>,
     ) -> Result<(), ContractError>;
+
+    /// Transfers the admin role of a registered token to a new address.
```

Comment:
```suggestion
    /// Transfers the admin role of a Stellar Classic Asset registered under the `token_id` from the corresponding token manager to the `new_admin`.
```
clarify this is for SAC